src/main/java/com/yahoo/bullet/storm/BulletStormConfig.java

```diff
@@ -62,16 +67,21 @@
      * Constructor that loads specific file augmented with defaults.
      *
      * @param file YAML file to load.
-     * @throws IOException if an error occurred with the file loading.
      */
-    public BulletStormConfig(String file) throws IOException {
-        // Load Bullet defaults
-        super(null);
-        // Load Bullet Storm settings
-        Config stormDefaults = new Config(file, DEFAULT_STORM_CONFIGURATION);
-        // Merge Storm settings onto Bullet defaults
-        merge(stormDefaults);
-        log.info("Bullet Storm merged settings:\n {}", getAll(Optional.empty()));
+    public BulletStormConfig(String file) {
+        this(new Config(file));
+    }
+
+    /**
+     * Constructor that loads the defaults and augments it with defaults.
+     *
+     * @param other The other config to wrap.
+     */
+    public BulletStormConfig(Config other) {
+        // Load Bullet and Storm defaults. Then merge the other.
+        super(DEFAULT_STORM_CONFIGURATION);
+        merge(other);
+        log.info("Merged settings:\n {}", getAll(Optional.empty()));
```

Comment:
I saw this change in your previous PRs. Will you delete those from your previous PRs?